content/docs/command-reference/add.md

```diff
@@ -379,41 +377,24 @@ outs:
     path: data.xml
 ```
 
-## Example: Transfer to remote storage
-
-When you have a large dataset in an external location, you may want to track it
-as if it was in your project, but without downloading it locally (for now). The
-`--to-remote` option lets you do so, while storing a copy
-[remotely](/doc/command-reference/remote) so it can be
-[pulled](/doc/command-reference/plots) later.
+[linked]:
+  /doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache
+[external cache]:
+  /doc/user-guide/managing-external-data#setting-up-an-external-cache
 
-Let's add the `data.xml` to our remote storage from the given remote location:
-
-```dvc
-$ dvc add https://data.dvc.org/get-started/data.xml -o data.xml \
-          --to-remote
-```
+## Example: Transfer to remote storage
 
-The only difference that dataset is transferred straight to remote, so DVC won't
-control the remote location you gave but rather continue managing your remote
-storage where the data is now on. The operation will still be resulted with an
-`.dvc` file:
+Use `--to-remote` to create a `.dvc` file for the operation without downloading
+data, transferring it directly to [remote storage] instead:
```

Comment:
Do you think it's worth mentioning the use case here? You may not have enough storage locally, but you want to pull it later on a machine with more space.

Reply:
Updated the motivation in both examples and reinstated everything else in the case of `add`. Linking to `import-url` for details stopped making sense (again).